### PR TITLE
Javascript javascript2 week2

### PR DESCRIPTION
**Pull request by RodrigoGuinsbergPinto**
Hi, in the first commit I added the wrong HTML file. 
I made a change to the file and committed again with 'correct files' description.

### CODE REVIEW

File: javascript/javascript2/week2/index2.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    <title>hyfBayHelpers</title>
+  </head>
+  <body>
+    <h1>HYF BAY</h1>
+    <main class="app">
+      <div class="input-boxes">
+        <h2>Our Products</h2>
+        <form class="form form-input">
+          <input id="searchInput" type="text" class="search-input" />
+          <input id="maxPrice" type="text" name="" class="max-price-input" />
```
Comment (annamourasobral) [inline javascript/javascript2/week2/index2.html]:
```suggestion
          <input id="maxPrice" type="number" name="" class="max-price-input" />
```
It's always nice to be specific in which kind of input you're asking from the user.

File: javascript/javascript2/week2/index2.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+          <label class="form-label">Search for product</label>
```
Comment (annamourasobral) [inline javascript/javascript2/week2/index2.html:21]:
```suggestion
          <label class="form-label">Search for product</label>
          <input id="searchInput" type="text" class="search-input" />
```
The label should be right before the input element that is referred to.

File: javascript/javascript2/week2/main.css [hunk 3]
```diff
@@ -0,0 +1,49 @@
+* {
```
Comment (annamourasobral) [inline javascript/javascript2/week2/main.css:1]: Good job on CSS!

File: javascript/javascript2/week2/main.js [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+searchInput.addEventListener("keyup", function () {
+  const inputValue = searchInput.value;
+  const filteredProducts = products.filter((product) =>
+    product.name.toLowerCase().includes(inputValue.toLowerCase())
+  );
+  if (!inputValue || inputValue === " ") {
+    productsList.innerHTML = "";
+    alert("Please enter product name!");
+    // renderProducts(products);
```
Comment (annamourasobral) [inline javascript/javascript2/week2/main.js]:
```suggestion
    renderProducts(products);
```
If you comment on this, the list of products is gone when you erase the input. The same happens on maxPrice.

File: javascript/javascript2/week2/script.js [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+  }
+  if (movie.rating >= 4) {
+    return (movie.tag = "Average");
+  } else {
+    return (movie.tag = "Bad");
+  }
+});
+// Using Ternary Operator
+// let rating =
+//     (movie.rating >= 7) ? "Good" : movie.rating >= 4 && movie.rating < 7 ? "Average" : "Bad";
```
Comment (annamourasobral) [inline javascript/javascript2/week2/script.js]: movie.rating >= 7 ? "Good" : movie.rating >= 7 ? "Average" : "Bad";

I think this is what you're looking for.
[This article](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator#conditional_chains),  will help you to understand clearly.

**Review by annamourasobral — CHANGES_REQUESTED** [review]
Hey, @RodrigoGuinsbergPinto 
Awesome work here! 
Besides us being both Brazilians, HYF's official language is English so I'll continue using it so everybody in the school can understand us. 😊
Let me know if you have any questions about the suggestions in Slack.
Keep going, and keep coding! 👩🏻‍💻

**Comment by alexchessmaster** [comment]
You're doing great!
I test it and it's working well

File: javascript/javascript2/week2/index2.html (hunk 1 above)
Comment (RodrigoGuinsbergPinto) [reply javascript/javascript2/week2/index2.html]: Of course : )  Thank you for pointing that out.

**Pull request merged**